src/HttpOverStream.NamedPipe/HttpOverStream.NamedPipe.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net462</TargetFramework>
```

Comment:
We really want to go back to targeting .net framework explicitly here?

Reply:
Don't have a choice, .netstandard / .net core PipeSecurity is broken.

Reply:
(that is one of the 2 additions I made for the Docker Desktop .net core POC)